addons/account/models/chart_template.py

```diff
@@ -1002,3 +1020,27 @@ def _load_translations(self, langs=None, companies=None):
                                     xml_id = f"account.{company.id}_{_xml_id}"
                                     translation_importer.model_translations[mname][fname][xml_id][lang] = value
         translation_importer.save(overwrite=False)
+
+    def _load_code_translations(self, accounts, langs=None):
+        """Loads translations for the accounts passed as argument from the code translations of the 'account' module.
+
+        :param accounts: the accounts to load the translations for
+        :type accounts: Model<account.account>
+        :param langs: the lang code to load the translations for.
+        :type langs: list[str]
+        """
+
+        langs = langs or [code for code, _name in self.env['res.lang'].get_installed()]
+        translations_dict = {
```

Comment:
Suggestion: Not really a fan of putting "dict" at the end of the variable because by reading the name of the variable we will supposed that it's a dict but for wathever reason the type change, it will be confusing